### ISSUE TITLE
Changing API

### ISSUE THREAD

**Issue by 142ayushkumar** [kaustubhhiware/c0derunR#41]
I think we should change to some other API. Many compilation errors using this one.

**Comment by kaustubhhiware** [comment]
Can we have a list of possible API's to use?
Or if in-house compilation and running is feasible or not. (if anyone prefers that)

**Comment by 142ayushkumar** [comment]
Some of API we can use are:
Hackerrank,repl

**Comment by 142ayushkumar** [comment]
@kaustubhhiware can you tell me more about in-house compilation?

**Comment by kaustubhhiware** [comment]
In-house compilation would mean we do not use an external API for compiling / running our code. In that approach, we'll have to check if that's possible in django, using some pip modules.

**Comment by Demfier** [comment]
Yeah, I was also thinking about it. If there is no such module in pip,
there is always the `os` module that can help. In any case, we'll have to
install the compilers for all the languages first in our server.
